Mersini_Kiara_hw2

kiara-mersini opened the pull request:
Added Home-Work 2 Folder

kochaika reviewed (changes requested):
You are deleting your own hw1
I can merge it but in this case you will not get points for hw1 -- you are removing it

kiara-mersini commented:
Hello,
What about the second homework? Will I get it merged?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Konstantin Chaika ***@***.***>
Sent: Monday, March 3, 2025 10:21:51 AM
To: kochaika/nand2tetris-hw-2025 ***@***.***>
Cc: Mersini, Kiara ***@***.***>; Author ***@***.***>
Subject: Re: [kochaika/nand2tetris-hw-2025] Mersini_Kiara_hw2 (PR #322)

You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

@kochaika requested changes on this pull request.

You are deleting your own hw1
I can merge it but in this case you will not get points for hw1 -- you are removing it

—
Reply to this email directly, view it on GitHub<https://github.com/kochaika/nand2tetris-hw-2025/pull/322#pullrequestreview-2653514640>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BPVVUZC4E3P6D6LBEFRIDRL2SQNK7AVCNFSM6AAAAABYEROR5SVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDMNJTGUYTINRUGA>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

kochaika commented:
> What about the second homework? Will I get it merged?

It you want to lose points for your hw1 -- sure
If not -- fix this PR

kiara-mersini commented:
Hello,
I fixed the second pull request, I added the previous deleted files. Is everything okay now?

________________________________
From: Konstantin Chaika ***@***.***>
Sent: Monday, March 3, 2025 10:49 AM
To: kochaika/nand2tetris-hw-2025 ***@***.***>
Cc: Mersini, Kiara ***@***.***>; Author ***@***.***>
Subject: Re: [kochaika/nand2tetris-hw-2025] Mersini_Kiara_hw2 (PR #322)

You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

What about the second homework? Will I get it merged?

It you want to lose points for your hw1 -- sure
If not -- fix this PR

—
Reply to this email directly, view it on GitHub<https://github.com/kochaika/nand2tetris-hw-2025/pull/322#issuecomment-2693810396>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BPVVUZBKBCZI4GLT3LFRQ232SQQTJAVCNFSM6AAAAABYEROR5SVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOJTHAYTAMZZGY>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

[kochaika]kochaika left a comment (kochaika/nand2tetris-hw-2025#322)<https://github.com/kochaika/nand2tetris-hw-2025/pull/322#issuecomment-2693810396>

What about the second homework? Will I get it merged?

It you want to lose points for your hw1 -- sure
If not -- fix this PR

—
Reply to this email directly, view it on GitHub<https://github.com/kochaika/nand2tetris-hw-2025/pull/322#issuecomment-2693810396>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BPVVUZBKBCZI4GLT3LFRQ232SQQTJAVCNFSM6AAAAABYEROR5SVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMOJTHAYTAMZZGY>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
